package Ch02;

public class C05ExceptionMain {

	public static void main(String[] args) {
		
		
		try {
//String str = null;
//str.toString();
	int [] arr = {10,20,30};
	arr[5] = 100;
////	
//	Animal bodong1 =new Dog();
//	Cat down = (Cat)bodong1;
//	
		}catch(Exception e) {System.out.println(e.getCause()+"예외처리!!!!");}
		
		
	}
}
